doc/introduction/photonic_hardware.rst

```diff
@@ -134,25 +134,27 @@ Cloud simulator
 ---------------
 
 In addition to submitting jobs to be run on quantum hardware, it is also possible to run jobs on
-the Xanadu Quantum Cloud simulator Simulon. The process is very similar to running jobs on hardware. You
+simulators via the Xanadu Quantum Cloud. The process is very similar to running jobs on hardware. You
 will need to configure your account, as described above, and submit a job via the ``RemoteEngine``,
-using ``"simulon"`` as the target instead of a specific chip:
+using a simulator as the target instead of a specific chip:
 
->>> eng = sf.RemoteEngine("simulon")
+>>> eng = sf.RemoteEngine("simulon_gaussian")
 >>> result = eng.run(prog)
 
 Simulon jobs can also be submitted asynchronously using ``eng.run_async``, or by submitting a
```

Comment:
```suggestion
Jobs to the ``simulon_gaussian`` simulator can also be submitted asynchronously using ``eng.run_async``, or by submitting a
```

Reply:
I actually deliberately changed this because in the near future other simulons will be available, like `simulon_jet`, and I figured it would be easier to imply there are multiple simulons now rather than change it again later when this becomes true.

Reply:
Got it! Maybe it's worth mentioning then that `simulon` is a general term for remote simulators.

Reply:
Good point, I made this change.